website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -104,10 +119,12 @@ By adding the annotation `helm.operator-sdk/upgrade-force: "True"` to the deploy
 apiVersion: example.com/v1alpha1
 kind: Nginx
 metadata:
-  name: example-nginx
+  name: nginx-sample
   annotations:
     helm.operator-sdk/upgrade-force: "True"
 spec:
+  # Default values copied from <project_dir>/helm-charts/nginx/values.yaml
+  ...
```

Comment:
This section is about the force annotation, so keep everything else as simple as possible to highlight what we're actually trying to discuss.
```suggestion
```